ts/navigation/TabNavigator.tsx

```diff
@@ -88,6 +98,20 @@ export const MainTabNavigator = () => {
             tabBarIcon: ({ color }) => <WalletTabIcon color={color} />
           }}
         />
+        {/* This item refers to the new "Scan" feature that allows
+      citizens to quickly manage a payment or signature request
+      from a printed paper and/or a third-party website.
+      It will be activated with a specific screen layout.
+      It differs from the existing "Scan QR Code" because
+      it's not limited to pagoPA payments. */}
+        {/* <Tab.Screen
```

Comment:
Instead of leaving commented code into the application, why do not we use a feature flag to disable it?

Reply:
As said by @Vangaorth, it would be best to avoid commented code. If this is a future task you can add a TODO with a Jira reference instead.